stack/base/before-notebook.d/40_prepare-aiida.sh

```diff
@@ -74,6 +74,9 @@ load_computer('${computer_name}').set_minimum_job_poll_interval(${job_poll_inter
 else
 
   # Migration will run for the default profile.
+  pgrep -af 'verdi.* daemon' && echo "ERROR: AiiDA daemon is already running!" && exit 1
+  ## clean up stale PID-files -> .aiida/daemon/circus-{profile-name}.pid
+  rm -f "/home/${NB_USER}/.aiida/daemon/circus-*.pid"
```

Comment:
Hmm, looks like this command doesn't actually remove the pid file? Let's try without the quotes.

```suggestion
  rm -f /home/${NB_USER}/.aiida/daemon/circus-*.pid
```